boards/xtensa/esp32/Kconfig.board

```diff
@@ -0,0 +1,9 @@
+# Kconfig - ESP32 board configuration
+#
+# Copyright (c) 2017 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_ESP32
+	bool "ESP32 Development Board"
+	depends on XTENSA
+	select BOARD_XTENSA
```

Comment:
Ugh, at some point we should just use y/n values of CONFIG_SIMULATOR_XTENSA instead of having both CONFIG_SIMULATOR_XTENSA and CONFIG_BOARD_XTENSA